src/game/index.js

```diff
@@ -1,24 +1,40 @@
 import {Subject} from 'rxjs';
 import {WAITING_PLAYERS} from './consts';
+import {createPlayers, getNextCard} from './util';
 
 
 export default class Game {
   state = {
     status: WAITING_PLAYERS,
-    players: [],
+    players: createPlayers(),
+    pile: [],
+    turnIndex: 0,
+    cardCount: 'K',
   };
 
   constructor() {
     this.subject = new Subject();
     this.observable = this.subject.asObservable();
   }
 
-  addPlayer = (name) => {
-    this.state.players.push({
-      name,
-    });
+  drawCard = () => {
+    const {turnIndex, players} = this.state;
+    const player = players[turnIndex];
+    const card = player.drawCard();
+    this.state.pile.push(card);
+
+    this.endTurn();
+  }
+
+  endTurn = () => {
+    const {turnIndex, players, cardCount} = this.state;
+
+    this.state.turnIndex = (turnIndex + 1) % players.length;
+    this.state.cardCount = getNextCard(cardCount);
+
     this.subject.next(this.state);
-  };
+  }
```

Comment:
This line should end with a semicolon. Did you run the linter?

Reply:
creo que al final no jejejeje